tools/x3gen.py

```diff
@@ -44,8 +39,8 @@ def __init__(self, ofile):
 
         self.options = capypdf.Options()
         self.fontfile = '/usr/share/fonts/truetype/noto/NotoSans-Regular.ttf'
-        self.imagefile = 'sampleimage.tif'
-        self.profile = 'UncoatedFOGRA29.icc'
+        self.imagefile = os.path.abspath('images/sampleimage.tif')
```

Comment:
Shouldn't this be `images/fogra_cmyk.tif`?

Reply:
Yes it should. One second.